docs/android/submit.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Android Submit Contribution"
+sidebar_label: "Submit Contribution"
+---
+
+## Submit Your First Contribution
+
+Congratulations on completing your first version! Now it's time to get feedback and prepare your work for real users. Follow the [GitHub Documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) to create a pull request (PR) from your fork.
+
+### Checklist for Submitting a PR
+
+When creating a PR, GitHub pre-fills the description with a checklist. Ensure you follow all the steps. Here's an extended checklist to help you:
+
+- ✅ **PR Description**: Provide a clear and complete description of your changes.
+- ✅ **Tests**: Add all necessary tests following our [testing guidelines](testing/introduction).
+- ✅ **Documentation**: Ensure your code is properly documented.
+- ✅ **UI Changes**: Include screenshots if the UI is modified.
+- ✅ **User Documentation**: If user documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/companion.home-assistant).
+- ✅ **Developer Documentation**: If this documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/developers.home-assistant/).
+- ✅ **Builds**: Verify that everything builds properly (app, automotive, wear) locally.
+- ✅ **Best Practices**: Follow the [best practices](bestpractices).
+- ✅ **Code Style**: Adhere to the [code style](codestyle).
+- ✅ **Linting**: Ensure no lint issues are introduced ([linter](linter)).
+
+### Opening a Draft PR
+
+If your PR isn't ready for official review but you'd like feedback, open it in **draft mode**. This will still trigger the CI on every commit and is especially useful when working on CI-related changes.
```

Comment:
CI is only automatically triggered if you have contributed once. If you're a new contributor, it needs to be approved by a maintainer every time. As this docs page is probably mostly read by first-time contributors, this feels misleading to me. The main developer docs mention this.

Reply:
This comment is still relevant, despite the outdated label.

Reply:
Updated